django_airbrake/middleware.py

```diff
@@ -8,4 +8,8 @@ def __init__(self):
 
     def process_exception(self, request, exception):
         if hasattr(settings, 'AIRBRAKE') and not settings.AIRBRAKE.get('DISABLE', False):
-            self.client.notify(exception=exception, request=request)
+            try:
+                self.client.notify(exception=exception, request=request)
+            # Ignore failures from issues like rate limiting
+            except:
+                pass
```

Comment:
I may be wrong here, but my understanding from [the documentation](https://docs.djangoproject.com/en/1.9/topics/http/middleware/#process-exception) was that returning None has the desired effect of raising the original exception as intended by leaving it to Django's exception processing.